pandas/tests/groupby/test_apply.py

```diff
@@ -851,3 +851,39 @@ def test_apply_function_returns_non_pandas_non_scalar(function, expected_values)
     result = df.groupby("groups").apply(function)
     expected = pd.Series(expected_values, index=pd.Index(["A", "B"], name="groups"))
     tm.assert_series_equal(result, expected)
+
+
+def test_apply_fast_slow_identical():
```

Comment:
see if we have similar fast/slow tests; place these by them (if not ok).

Reply:
There are a few tests further up the file wich affect similar code paths, e.g. `test_fast_apply` or `test_group_apply_once_per_group` which seems a suitable place